sote/game/scenes/game.lua

```diff
@@ -1245,6 +1509,11 @@ function gam.draw()
 			map_mode_bar_layout:next(UI_STYLE.square_button_large, UI_STYLE.square_button_large), gam.map_mode_data['biomes'][3]) then
 		gam.click_callback = callback.update_map_mode(gam, "biomes")
 	end
+	if ut.icon_button(
+			ASSETS.icons[gam.map_mode_data['terrain'][2]],
```

Comment:
These map mode datas will need to be reworked to be properly typed.
I don't think we have a tracking issue for it :c